src/Schema/Elements/AnyOf.php

```diff
@@ -98,7 +98,7 @@ public function complete($value, Nette\Schema\Context $context)
 			$expecteds = implode('|', array_unique($expecteds));
 			$context->addError(
 				'The option %path% expects to be %expected%, %value% given.',
-				__CLASS__ . ':unexpected',
+				Nette\Schema\ErrorCodes::UNEXPECTED_VALUE,
```

Comment:
I think we can (re)use `ValidationException` here, ie. `ValidationException::ERROR_UNEXPECTED_VALUE`, because it is related to `ValidationException::getErrors()`

Reply:
Makes sense, done